components/engine_rocks/src/properties.rs

```diff
@@ -619,6 +621,26 @@ pub fn get_range_stats(
     })
 }
 
+pub fn get_all_table_stats(engine: &crate::RocksEngine, cf: &str) -> EngResult<RangeStats> {
+    let collection = engine.get_properties_of_all_tables_cf(cf)?;
```

Comment:
Nit, read properties of all sst files may take some time and memory, would be nice if we can record this event in metrics.